tests/positive/Format.juvix

```diff
@@ -297,13 +297,22 @@ module Traits;
   type Show A := mkShow {show : A → String};
 
   instance
-  showStringI : Show String := mkShow (show := id);
+  showStringI : Show String :=
```

Comment:
Does it make sense to you why the juvix formatter would break lines on these code blocks given that we increased the ribbon width?

Reply:
The explanation is that after `:=` we break if the body does not fit in a line. The formatter _always_ inserts newlines in the new named application syntax. I.e. you'll never see `mkShow@{show := id}`.
I think there are 3 sensible ways to format the named application syntax.
1. [what we currently do] Always break.
2. Break iff there are more than 1 argument.
3. Break only if it does not fit in a line.

I'm fine with both 1 and 2
If we went, with 2 my expectation is that there would not be a linebreak after `:=`.

Reply:
I see - I'm also happy with what we currently do. Thanks for the explanation.